### PR TITLE
Add a fallback mechanism when no intact basals

### PR DESCRIPTION
**Pull request by wizmer**
Respect the logic from:
https://bbpcode.epfl.ch/source/xref/platform/BlueRepairSDK/BlueRepairSDK/src/repair.cpp#658

### CODE REVIEW

File: tests/test_repair.py [hunk 1]
```diff
@@ -57,7 +57,6 @@ def test__find_intact_sub_trees():
     obj._fill_repair_type_map()
     assert_raises(Exception, obj._find_intact_sub_trees)
```
Comment (asanin-epfl) [inline tests/test_repair.py:58]: I am curious why this line passes if you dropped this exception `Exception('No intact basal dendrites !')`

Comment (wizmer) [reply tests/test_repair.py:58]: Lol, it was failing for a different reason. A list was passed were a numpy array was expected. Thanks for spotting this !

File: tests/test_repair.py [hunk 2]
```diff
@@ -53,10 +53,8 @@ def test__find_intact_sub_trees():
     assert_equal(len(obj._find_intact_sub_trees()), 2)
 
     obj = Repair(SIMPLE_PATH)
-    obj.cut_leaves = [[0, 0, 0]]
+    obj.cut_leaves = np.array([[0, 0, 0]])
     obj._fill_repair_type_map()
```
Comment (asanin-epfl) [inline tests/test_repair.py:57]: Does it make sense to keep these 3 lines at all? We don't assert anything after them.

Comment (wizmer) [reply tests/test_repair.py]: Yes, I've removed them. Good point.

**Comment by wizmer** [comment]
Thank you !

**Comment by asanin-epfl** [comment]
np

**Pull request merged**